test/snapper_dbus_escape_test.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env rspec
```

Comment:
I'm using one test file for each class. Also that approach is used on [SnapperTree](https://github.com/aschnell/yast-snapper/blob/master/test/snapper_tree_test.rb test).